spec/numberformat.html

```diff
@@ -346,6 +365,13 @@
 
       <emu-alg>
         1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
+        1. Let _this_ be the receiver.
+        1. NOTE: The following step and nested steps implement legacy compatibility semantics
```

Comment:
this is the first time I see an arbitrary link in an ecma spec, do we really need to link to the bug in the algo?